### ISSUE TITLE
Bug: Toggle Group on Contributor Scatter Chart has no label

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#3881]
The toggle group that houses the different types of PRs for the contributor distribution in the contributor chart is missing a label.

![CleanShot 2024-08-07 at 00 22 05](https://github.com/user-attachments/assets/0405d755-4174-421b-943a-74fe7859fd6f)

I came across this while implementing #3863
